scenarios/19a_The_Woods_of_Okladia.cfg

```diff
@@ -52,7 +52,7 @@
         {GOLD 240 280 320}
         {INCOME 4 5 6}
         [ai]
-            recruitment_pattern=fighter,fighter,healer,fighter
+            recruitment_pattern=fighter,1,2,1,fighter,healer,fighter
```

Comment:
What does this do?

Reply:
the numbers represent levels of units to recruit; see conversation with @celticminstrel in screenshot in wesnoth/wesnoth#6006

Reply:
Update: `wmllint` now recognizes the numbers after wesnoth/wesnoth@33fbc52